mento-protocol/mento-analytics-api
# feat: improve price fetching

**Description**

This change adds the following:
- Adds a cache warming service that will fetch and cache the API data on startup and every 3 hours
- Increased RPC timeout for UNIV3 position fetcher
- Optimized CMC API integration

The 3-hour cache was based on recent reserve site traffic + the fact that the data does not need to be so fresh. We can think about increasing this to reduce RPC calls. 

**Related Issue**
https://github.com/mento-protocol/reserve-site/issues/123

**How to test**
This can be tested locally or by hitting the deployed API endpoints. We should check to make sure the values being returned by the API match what is displayed on the [current reserve site](https://reserve.mento.org/).

Deployed app logs [here](https://console.cloud.google.com/run/detail/us-central1/mento-analytics-api/logs?project=mento-prod)

Docs with endpoints [here](https://mento-analytics-api-12390052758.us-central1.run.app/docs)
